pageserver/src/tenant/timeline.rs

```diff
@@ -884,7 +891,12 @@ impl Timeline {
     pub async fn shutdown(self: &Arc<Self>, freeze_and_flush: bool) {
         debug_assert_current_span_has_tenant_and_timeline_id();
 
+        // Signal any subscribers to our cancellation token to drop out
+        tracing::info!("Cancelling CancellationToken");
+        self.cancel.cancel();
+
         // prevent writes to the InMemoryLayer
+        tracing::info!("Waiting for WalReceiverManager...");
```

Comment:
```suggestion
        tracing::debug!("Waiting for WalReceiverManager...");
```

Reply:
I think the normal shutdown_tasks logging will also apply to this if it would take long, so these might not be needed at all.